nixos/modules/services/torrent/transmission.nix

```diff
@@ -249,6 +249,29 @@ in
           For instance, SSH sessions may time out more easily.
         '';
       };
+
+      dhtBootstrapNodes = mkOption {
+        type = with types;
+          listOf (submodule {
+            options = {
+              host = mkOption { type = str; description = "Hostname or IP address."; };
+              port = mkOption { type = port; description = "Port number."; };
+            };
+          });
+        default = [ ];
+        description = "List of nodes from which to bootstap into the DHT.";
```

Comment:
The description should be wrapped by an `mdDoc` nowadays.

Reply:
Those are just strings, not mddoc documents. I'd rather just wait for this "transition period" to pass.

Reply:
@ehmry, yet `mdDoc` is required, have you seen the [manual build failure](https://github.com/NixOS/nixpkgs/actions/runs/3378990469/jobs/5609953849)? Same applies for `host` and `port` options.

Reply:
> yet mdDoc is required

offtopic, but this migration looks crazy to me.